[2023-09-13T07:38:27.759+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-09-13T07:38:27.794+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-09-13T07:38:27.796+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:27.851+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-27 00:00:00+00:00
[2023-09-13T07:38:27.870+0000] {standard_task_runner.py:57} INFO - Started process 85 to run task
[2023-09-13T07:38:27.940+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpcdxm_gsp']
[2023-09-13T07:38:27.944+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask extract_data
[2023-09-13T07:38:28.259+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:28.515+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-09-13T07:38:28.999+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.056+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230827T000000, start_date=20230913T073827, end_date=20230913T073829
[2023-09-13T07:38:29.139+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.175+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:08.498+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.513+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-09-15T11:31:08.514+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:08.547+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-27 00:00:00+00:00
[2023-09-15T11:31:08.560+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpreez9o8s']
[2023-09-15T11:31:08.567+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask extract_data
[2023-09-15T11:31:08.559+0000] {standard_task_runner.py:57} INFO - Started process 332 to run task
[2023-09-15T11:31:08.666+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-27T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:08.780+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-09-15T11:31:09.069+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:09.082+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230827T000000, start_date=20230915T113108, end_date=20230915T113109
[2023-09-15T11:31:09.109+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:09.129+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
